command/cp.go

```diff
@@ -113,6 +119,14 @@ var copyCommandFlags = []cli.Flag{
 		Value:   defaultPartSize,
 		Usage:   "size of each part transferred between host and remote server, in MiB",
 	},
+	&cli.StringFlag{
+		Name:  "source-region",
+		Usage: "connect to a specific region of the remote object storage service",
+	},
+	&cli.StringFlag{
+		Name:  "region",
```

Comment:
Why do we need this flag? I still don't understand why do we need to "attach" source and destination sessions.

Reply:
when running in batch mode, say `s5cmd run commands.txt` where `commands.txt` consists of 

`cp s3://bucket/region_X/ s3://bucket/region_Y/`
`cp s3://bucket/region_K/ s3://bucket/region_P/`

since we have buckets in four different regions, I thought maybe we should somehow specify those.

Reply:
since we 'infer' bucket regions and this flag has been removed, I think it is resolved :)